PCbuild/prepare_ssl.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+# -*- coding: utf8 -*-      for <2.7>/PC/VS9.0/build_ssl.py
```

Comment:
I firmly believe that a comment is needed as this encoding decl has been removed once already.  Not to mention that, with it removed, there will be no discernible issues with its removal *until* an updated OpenSSL is pushed to the buildbots.  Especially due to the fact this is in a seldom used corner of the Python build process.